workshops/memory game/README.md

```diff
@@ -0,0 +1,376 @@
+
+## The JavaScript
+
+Create a DOM [event-listener](https://developer.mozilla.org/en-US/docs/Web/API/EventListener) for the event [DOMContentLoaded](https://developer.mozilla.org/en-US/docs/Web/API/Document/DOMContentLoaded_event). All our JavaScript code will be inside the event-listener, which will be excuted after the content loaded.
+
+```javascript
+document.addEventListener('DOMContentLoaded', () => {
+  // code goes here!
+})
+```
+
+Inside the event-listener, create an array for the cards which we use for the game. Add two of each cards for matching the cards. The public links for the images are provided in the code below. If you downloaded the images or using your own images, then add the relative path to the images. Also name them as your wish.
+
+```javascript
+document.addEventListener('DOMContentLoaded', () => {
+  const cardArray = [
+    {
+      name: '1',
+      img: 'https://cloud-5ystxzer7.vercel.app/11.png'
+    },
+    {
+      name: '2',
+      img: 'https://cloud-5ystxzer7.vercel.app/22.png'
+    },
+    {
+      name: '3',
+      img: 'https://cloud-5ystxzer7.vercel.app/33.png'
+    },
+    {
+      name: '4',
+      img: 'https://cloud-5ystxzer7.vercel.app/44.png'
+    },
+    {
+      name: '5',
+      img: 'https://cloud-5ystxzer7.vercel.app/55.png'
+    },
+    {
+      name: '6',
+      img: 'https://cloud-5ystxzer7.vercel.app/06.png'
+    },
+    {
+      name: '1',
+      img: 'https://cloud-5ystxzer7.vercel.app/11.png'
+    },
+    {
+      name: '2',
+      img: 'https://cloud-5ystxzer7.vercel.app/22.png'
+    },
+    {
+      name: '3',
+      img: 'https://cloud-5ystxzer7.vercel.app/33.png'
+    },
+    {
+      name: '4',
+      img: 'https://cloud-5ystxzer7.vercel.app/44.png'
+    },
+    {
+      name: '5',
+      img: 'https://cloud-5ystxzer7.vercel.app/55.png'
+    },
+    {
+      name: '6',
+      img: 'https://cloud-5ystxzer7.vercel.app/06.png'
+    }
+  ]
+})
+```
+
+Now we are going to creating our game board.
+
+## Game Board
+
+Pick up the `div` element with `class = "board"` using [querySelector](https://developer.mozilla.org/en-US/docs/Web/API/Document/querySelector) and define it as `board`. Create a function `createBoard()` and loop over through elements in `cardArray` using `for` loop and add cards to our game board.
+
+Create a constant for our placeholder image. We are going to use it many times in the project.
+
+```javascript
+const placeholder = 'https://cloud-5ystxzer7.vercel.app/7placeholder.png'
+```
+
+Create a `img` element using `document.createElement` for the every card. Using `setAttribute()`, add `src` and `data-id` attributes to the image. Let's first set the `src` attribute to the placeholder image (i.e, let's assume the placeholder image as reversed card). The link to the placeholder image is given in the code, otherwise add the relative path to the image.
+
+![place holder image](https://cloud-5ystxzer7.vercel.app/7placeholder.png)
```

Comment:
This image appears to be here without any context. Can you either give some context here or move it earlier in the workshop?